src/main/resources/validationrules/core/hearing-role-judicial-global.drl

```diff
@@ -39,7 +39,11 @@ import function uk.gov.hmcts.reform.orgrolemapping.domain.service.RequestMapping
    "CIVIL Deputy District Judge-Fee-Paid","CIVIL Deputy District Judge - Sitting in Retirement-Fee-Paid",
    "CIVIL District Judge-Salaried","CIVIL Recorder-Fee-Paid","CIVIL Senior Circuit Judge-Salaried",
    "CIVIL High Court Judge-Salaried","CIVIL Designated Civil Judge-Salaried","CIVIL Presiding Judge-Salaried",
-   "CIVIL Resident Judge-Salaried"))
```

Comment:
Here an improvement can be placed.
as per current confluence any valid ET judge we have to create hearing role, instead of adding all the JOH names, this can be added like below(this will applicable to CIVIL and PUBLICLAW)
replace line 29 to 46 with
$joh: JudicialOfficeHolder(office in ("SSCS Judge", "Family Judge") || 
   jurisdiction in ("PUBLICLAW", "CIVIL", "EMPLOYMENT"))

Reply:
NB: Current Confluence for `"hearing-viewer"` roles for ET say: "Any Employment JOH", for Civil say: "Any valid Judicial Appointment/Role", for Family Public Law say :"Any Judicial Appointment".